src/test/java/tests/Smoke.java

```diff
@@ -0,0 +1,91 @@
+package tests;
+
+import java.util.concurrent.TimeUnit;
+import common.AbstractComponent;
+import io.github.bonigarcia.wdm.WebDriverManager;
+import org.openqa.selenium.*;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.interactions.Actions;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+import pages.*;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+
+public class Smoke {
+
+    WebDriver driver;
+    WebDriverWait wait;
+    /**
```

Comment:
this comment is incorrect, no constructer initialized after method. please remove